### ISSUE TITLE
Split regions if they are "split" by a separator

### ISSUE THREAD

**Issue by wrznr** [OCR-D/ocrd_segment#27]
![image](https://user-images.githubusercontent.com/26707219/72509648-41258480-3848-11ea-9a1b-0c485ba9c01b.png)

**Comment by bertsky** [comment]
Yes, I wonder what Tesseract is thinking when it does this. Such bad manners!

**Comment by bertsky** [comment]
Technically, what exactly would you propose to do? Calculate the point-set difference of the polygons, and then look at the resulting ~~`interior` sets~~ multipolygon as a sequence?

**Comment by bertsky** [comment]
Do you have an example where clipping is not enough to handle this?

**Comment by wrznr** [comment]
NO. It's just cosmetics.

**Comment by wrznr** [comment]
I thought about using https://shapely.readthedocs.io/en/stable/manual.html#splitting

**Comment by bertsky** [comment]
> I thought about using https://shapely.readthedocs.io/en/stable/manual.html#splitting

Yes, that would work.

But there are other issues besides coordinates:
- new IDs (append suffix to both, or just one)
- fix/update reading order (insert at same position, or leave out)
- splitting up existing text content or lines/words
